src/main/java/bio/terra/cli/businessobject/resource/BqDataset.java

```diff
@@ -183,4 +185,9 @@ public String getProjectId() {
   public String getDatasetId() {
     return datasetId;
   }
+
+  public int getTableCount() {
```

Comment:
nit: method description that notes this is not a simple getter; rather, it queries the cloud for more info